### ISSUE TITLE
Create issue template and update readme to guide users on where and how to open icons-related issues

### ISSUE THREAD

**Issue by aminalhazwani** [FirefoxUX/icons#68]
We have issues displaced across the two repos, this one and the `/photon-icons` one, should we have them all in just one repo? We could use this tool to move them easily https://github.com/google/github-issue-mover or something else

**Comment by bwinton** [comment]
Well, I feel like there are a couple of different purposes. This repo should have issues about the website, the tokens should have issues about adding or removing tokens…

Maybe we could have an informational form for people when they file a bug, instead?

**Comment by aminalhazwani** [comment]
Yes, I agree, let me rephrase the issue!
